packages/components/datagrid/src/js/datagrid.controller.js

```diff
@@ -4,7 +4,7 @@ import { hasProperty } from './util';
 
 import template from './datagrid.html';
 
-const cssSorted = 'oui-datagrid__header_sorted';
+const cssSorted = 'oui-datagrid__cell_sorted';
```

Comment:
Is there a reason that this classname has changed ?
The style is still applied to `oui-datagrid__header_sorted`.

Reply:
No, just a mistake